docs/source/open-software-summer-school/2026/large-data.md

```diff
@@ -0,0 +1,66 @@
+(track-large-array-data-2026)=
+# Track: Large Array Data
+
+In many areas of research, large array datasets have become important and wide-spread. Examples include whole-organ images (e.g. light-sheet, HiP-CT), Neuropixels data, and long behavioural videos — to name just a few. Processing, analyzing and visualising these datasets is made significantly more complex and slow because they do not fit into a typical computer's memory and take up a lot of disk space.
+
+The use of compression algorithms and "chunked" file formats (which favour parallel processing) enable us to deal with these challenges effectively, but doing this well requires careful consideration of the trade-offs between read and write speeds, disk space used, and portability.
+
+This track will cover the necessary context needed to understand the computational challenges posed by large array datasets, and how to address these challenges using open-source software tools. All sessions will include a significant hands-on component.
+
+::: {admonition} Target audience
+:class: note
+
+This course is designed for researchers and students interested in learning about open-source tools for processing large array data with Python. Participants are encouraged to bring their own data to experiment with, but example data will be provided.
```

Comment:
Should we make it clearer that it is for people who have, or expect to acquire large datasets? Currently "an immediate use case" is one of our selection criteria.